docs/kqueen.cli_gcloud_examples.rst

```diff
@@ -0,0 +1,137 @@
+Kqueen and Google cloud from scratch
+------------------------------------
+
+Create the Google Cloud account and start with `gcloud`:
```

Comment:
i prefer to mark each step with number
u may use '#'

Reply:
I don't think it's a good idea. This is not step by step guide and some steps can be mixed with others. This is rarely seen in other documentations...